app/controllers/users_controller.rb

```diff
@@ -22,8 +26,55 @@ def create
     end
   end
 
+  def edit
+  end
+
+  def update
+    if @user.update_attributes user_params
+      flash[:success] = t ".prupdate"
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    if @user.destroy
+      flash[:success] = t ".success"
+      redirect_to users_path
+    else
+      flash[:danger] = t "static_pages.home.usernotfound"
+      redirect_to users_path 
+    end
+  end
+
   private
     def user_params
       params.require(:user).permit :name, :email, :password, :password_confirmation
     end
+    
+    def logged_in_user
```

Comment:
To avoid code repetition and for reusing purpose, you should move this method to the application controller